apparelo/apparelo/doctype/dc/dc.js

```diff
@@ -4,6 +4,11 @@
 frappe.ui.form.on('DC', {
 	onload: function(frm) {
 		frm.set_value("company",frappe.defaults.get_default("company"));
+		frappe.db.get_list("Address",{filters:{is_primary_address: 1, is_shipping_address: 1}}).then(data => {
```

Comment:
remove 'is_primary_address' filter...

Reply:
Resolved by this https://github.com/aerele/apparelo/pull/182/commits/8302d6988855b71cd9f14c0142edd5e928b1c337

Reply:
👍🏽